spring-cloud-gcp-starters/spring-cloud-gcp-starter-storage/pom.xml

```diff
@@ -26,5 +26,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-storage</artifactId>
 		</dependency>
```

Comment:
See the issue. We have a problem exactly in the servlet-based application context